automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/NotebookPage.scala

```diff
@@ -84,4 +87,10 @@ class NotebookPage(override val url: String)(override implicit val authToken: Au
     await condition (!cellsAreRunning, timeoutSeconds)
     cellOutput(cell)
   }
+
+  def terminateKernelAndExit(): Unit = {
+    click on fileMenu
+    click on (await enabled closeAndWaitSelection)
```

Comment:
enabled or visible? enabled WebElement that is present in DOM could be invisible if page has not finished with rendering.

Reply:
Tests are run headlessly - does "visible" make sense in that case?